test/Utils.js

```diff
@@ -1059,5 +1059,106 @@ describe("#Utils", () => {
         "tokenQty"
       ])
     })
+
+    it("should process problematic utxos", async () => {
```

Comment:
I added this new test case, based on the UTXOs that were causing the bug for Pete. This new test case will ensure we don't see this bug again.